automation/run-tests.sh

```diff
@@ -74,38 +74,23 @@ function install_nmstate {
 function run_tests {
     if [ $TEST_TYPE == $TEST_TYPE_ALL ] || \
        [ $TEST_TYPE == $TEST_TYPE_FORMAT ];then
-        if [[ $CONTAINER_IMAGE == *"centos"* ]]; then
-            echo "Running formatter in $CONTAINER_IMAGE container is not " \
-                 "support yet"
-        else
-            container_exec 'tox -e black'
-        fi
+        container_exec 'tox -e black'
     fi
 
     if [ $TEST_TYPE == $TEST_TYPE_ALL ] || \
        [ $TEST_TYPE == $TEST_TYPE_LINT ];then
-        if [[ $CONTAINER_IMAGE == *"centos"* ]]; then
-            echo "Running unit test in $CONTAINER_IMAGE container is not " \
-                 "support yet"
-        else
-            container_exec 'tox -e flake8,pylint'
-        fi
+        container_exec 'tox -e flake8,pylint'
     fi
 
     if [ $TEST_TYPE == $TEST_TYPE_ALL ] || \
        [ $TEST_TYPE == $TEST_TYPE_UNIT_PY36 ];then
-        if [[ $CONTAINER_IMAGE == *"centos"* ]]; then
-            echo "Running unit test in $CONTAINER_IMAGE container is not " \
-                 "support yet"
-        else
-            container_exec 'tox -e py36'
-        fi
+        container_exec 'tox -e py36'
     fi
 
     if [ $TEST_TYPE == $TEST_TYPE_ALL ] || \
        [ $TEST_TYPE == $TEST_TYPE_UNIT_PY37 ];then
         if [[ $CONTAINER_IMAGE == *"centos"* ]]; then
-            echo "Running unit test in $CONTAINER_IMAGE container is not " \
+            echo "Running python 3.7 unit test in $CONTAINER_IMAGE container i not " \
```

Comment:
There is a typo, it now says "i not" instead of "is not".

Reply:
Done, thanks!